docs/getting-started/usage.md

```diff
@@ -1,64 +1,591 @@
-# Usage
+# Step-by-step guide
 
-Chart.js can be used with ES6 modules, plain JavaScript, and module loaders.
+Follow this guide to get familiar with all major concepts of Chart.js: chart types and elements, datasets, customization, plugins, components, and tree-shaking. Don't hesitate to follow the links in the text.
 
-## Creating a Chart
+We'll build a Chart.js data visualization with a couple of charts from scratch:
 
-To create a chart, we need to instantiate the `Chart` class. To do this, we need to pass in the node, jQuery instance, or 2d context of the canvas of where we want to draw the chart. Here's an example.
+![result](./usage-8.png)
+
+## Build a new application with Chart.js
+
+In a new folder, create the `package.json` file with the following contents:
+
+```json
+{
+  "name": "chartjs-example",
+  "version": "1.0.0",
+  "license": "MIT",
+  "scripts": {
+    "dev": "parcel src/index.html",
+    "build": "parcel build src/index.html"
+  },
+  "devDependencies": {
+    "parcel": "^2.6.2"
+  },
+  "dependencies": {
+    "@cubejs-client/core": "^0.31.0",
+    "chart.js": "^4.0.0"
+  }
+}
+```
+
+Modern front-end applications often use JavaScript module bundlers, so we’ve picked [Parcel](https://parceljs.org) as a nice zero-configuration build tool. We’re also installing Chart.js v4 and a JavaScript client for [Cube](https://cube.dev/?ref=eco-chartjs), an open-source API for data apps we’ll use to fetch real-world data (more on that later).
+
+Run `npm install`, `yarn install`, or `pnpm install` to install the dependencies, then create the `src` folder. Inside that folder, we’ll need a very simple `index.html` file:
 
 ```html
-<canvas id="myChart" width="400" height="400"></canvas>
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>Chart.js example</title>
+  </head>
+  <body>
+    <!-- <div style="width: 500px;"><canvas id="dimensions"></canvas></div><br/> -->
+    <div style="width: 800px;"><canvas id="acquisitions"></canvas></div>
+
+    <!-- <script type="module" src="dimensions.js"></script> -->
```

Comment:
should this script be removed from the example?

Reply:
Not really! Later in the guide, there's a step where these lines are supposed to be uncommented: https://github.com/chartjs/Chart.js/pull/10816/files/5218ca1ca4560d5206dfc0adb44e91564905d1f6#diff-746af86e15720621e0b7e6229e4f4ca9beba4db50c001007cbba9317f5379479R257